src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -763,11 +763,34 @@ def make_organoids(self, entry):
     }
 
     def make_sample(self, sample):
-        entity_type = one(sample['entity_type']) if isinstance(sample['entity_type'], list) else sample['entity_type']
-        return {
-            "sampleEntityType": self.sample_entity_types[entity_type],
-            **self.sample_entity_make_functions[entity_type](self, sample)
-        }
+        if isinstance(sample['entity_type'], list):
+            # Run make_<type>() for each sample['entity_type'] and merge result into sample_dict
+            sample_dict = {'sampleEntityType': []}
```

Comment:
I think this is still wrong. On `/repository/samples`, we don't want the values to be a list, correct?

Also note that `defaultdict` is your friend when imperatively building a dictionary. When translating between data structures, I also find nested comprehensions to be surprisingly concise and simple.